README.md

```diff
@@ -170,7 +168,7 @@ public function getJsonApi()
 }
 ```
 
-The description says it very clear: if you want a jsonApi section in your response, then create a new `JsonApi` object.
+The description says it very clear: if you want a JsonApi section in your response, then create a new `JsonApi` object.
```

Comment:
The usage of "jsonApi" is intentional here because the key in the JSON document is "jsonApi".